src/KM_Controls.pas

```diff
@@ -1702,15 +1725,7 @@ procedure TKMControl.DoClick(X,Y: Integer; Shift: TShiftState; Button: TMouseBut
       fTimeOfLastClick := TimeGet;
 
     if Assigned(fOnClickShift) then
-    begin

```

Comment:
This code is not needed anymore, as we are adding button to Shift in KM_FormMain unit, just at the start of MouseUp event processing